plugins/woocommerce/src/Admin/API/OnboardingThemes.php

```diff
@@ -249,9 +249,69 @@ public function get_recommended_themes( $request ) {
 			);
 		}
 
+		$coreThemes = array(
+			array(
+				'name'           => 'Twenty Twenty-Four',
+				'price'          => 'Free',
+				'color_palettes' => array(
+					array(
+						'title'     => 'Black and white',
+						'primary'   => '#FEFBF3',
+						'secondary' => '#7F7E7A',
+					),
+					array(
+						'title'     => 'Brown Sugar',
+						'primary'   => '#EFEBE0',
+						'secondary' => '#AC6239',
+					),
+					array(
+						'title'     => 'Midnight',
+						'primary'   => '#161514',
+						'secondary' => '#AFADA7',
+					),
+					array(
+						'title'     => 'Olive',
+						'primary'   => '#FEFBF3',
+						'secondary' => '#7F7E7A',
+					),
+				),
+				'total_palettes' => 0,
+				'slug'           => 'twentytwentyfour',
+				'thumbnail_url'  => 'https://i0.wp.com/themes.svn.wordpress.org/twentytwentyfour/1.0/screenshot.png',
+				'link_url'       => 'https://woo.local/wp-admin/themes.php?theme=twentytwentyfour',
```

Comment:
I believe this should point to `https://wordpress.com/theme/twentytwentyfour`?

Reply:
Nice catch, thanks! Since this is the implementation on core, updated to https://wordpress.org/themes/twentytwentyfour/ instead of `.com`